lib/util/util_abort_gnu.c

```diff
@@ -313,10 +315,12 @@ void util_abort__(const char * file , const char * function , int line , const c
     }
     chmod(filename, 00644); // -rw-r--r--
     free(filename);
+    pthread_mutex_unlock(&__abort_mutex);
+  } else {
+    /* Failure in util_abort. Print basic error message and exit. */
+    fprintf(stderr, "Error while handling error. Exiting. File: %s Function: %s, Line: %d\n", file, function, line);
```

Comment:
This is fine, but we are not printing the content of the `va_list`. One option could be to move the lines 267-275 outside the if and make it print to a string instead. Then, if we can lock the mutex, we write the string to file, otherwise we display it on stderr

Reply:
I'll just repeat those lines needed for diplaying the error if that is ok.